src/main/java/com/ksm/wstbackoffice/service/IWarehouseService.java

```diff
@@ -0,0 +1,12 @@
+package com.ksm.wstbackoffice.service;
+
+import com.ksm.wstbackoffice.dto.WarehouseDto;
+
+import java.util.List;
+
+public interface IWarehouseService {
```

Comment:
Please do not use Interface in your class.
We will do it for all services in the future.

Reply:
fixed